whiskey.yml

```diff
@@ -3,7 +3,10 @@ Build:
 - Version:
     Path: Carbon.IIS\Carbon.IIS.psd1

```

Comment:
Version not bumped?

Reply:
We haven't released 1.0.0 of this module yet. It was extracted from Carbon.